solid/objects.py

```diff
@@ -0,0 +1,535 @@
+"""
+Classes for OpenSCAD builtins
+"""
+from .solidpython import OpenSCADObject
```

Comment:
Everything after this line is just directly copied with no changes from `solidpython.py`